src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -397,39 +421,39 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock \
                        eclipse/che:<version> [COMMAND]
 ----
 
-To activate jpda suspend mode for debugging Che server initialization, in the `che.env`:
+To activate jpda suspend mode for debugging Che server initialization, in the `che.env` file:
```

Comment:
* what's 'jpda'? and should it be written that way?
* verb missing after the comma?

Reply:
jpda -> JPDA
Changed to "To activate JPDA suspend mode for debugging Che server initialization, add the following in the `che.env` file:"